tests/unit/Models/MetadataTraitTest.php

```diff
@@ -679,6 +680,8 @@ public function knownSideProvider()
             [TestMorphOneSource::class, false],
             [TestMorphOneSourceAlternate::class, false],
             [TestMorphTarget::class, true],
+            [TestMorphTarget::class, true],
+            [TestMorphManySourceWithUnexposedTarget::class, false]
```

Comment:
As TestMorphManySourceWithUnexposedTarget has no polymorphic relations with exposed models, it should act as a monomorphic model